hack/deploy-custom-nto.sh

```diff
@@ -57,21 +64,8 @@ wait_for_updated_tuned_pods() {
   done
 }
 
-nto_run_tuned_tests() {
-  # test-e2e usually take 30min
-  make test-e2e
-
-  CLUSTER=mcp-only make cluster-deploy-pao
-  # e2e-gcp-pao	usually take 40min
-  make pao-functests
-
-  # e2e-gcp-pao-updating-profile usually take 4-5h
-  make pao-functests-updating-profile
-}
-
 nto_prepare_image
 cvo_scale 0
 nto_deploy_custom
 wait_for_updated_tuned_pods
-nto_run_tuned_tests
 cvo_scale 1
```

Comment:
`cvo_scale 1` needs to be removed, otherwise CVO will bring back its own version of NTO.  This was fine after the nto_run_tuned_tests were done, but we cannot leave it here now.